### PR TITLE
Added browser validation

### PR DESCRIPTION
**Pull request by KalibRx**
#### What does it implement/fix? Explain your changes.
Adds back browser validation of the XSS payloads with definitive alert box verification.

#### Where has this been tested?
Python Version: 3.6
Operating System: Linux/Windows

#### Does this close any currently open issues? 
Not sure

#### Does this add any new dependency?
No

#### Any other comments you would like to make?
Please verify I did not remove too much in adding the validation back.

#### Some Questions
- [ ] I have documented my code.
- [X] I have tested my build before submitting the pull request.

### CODE REVIEW

**Comment by KalibRx** [comment]
It's currently failing the build on Travis due to lack of geckodriver on the build testing box

File: modes/scan.py [hunk 1]
```diff
@@ -94,30 +95,20 @@ def scan(target, paramData, encoding, headers, delay, timeout, skipDOM, find, sk
...
-                if not efficiencies:
-                    for i in range(len(occurences)):
-                        efficiencies.append(0)
-                bestEfficiency = max(efficiencies)
-                if bestEfficiency == 100 or (vect[0] == '\\' and bestEfficiency >= 95):
-                    logger.red_line()
-                    logger.good('Payload: %s' % loggerVector)
-                    logger.info('Efficiency: %i' % bestEfficiency)
-                    logger.info('Confidence: %i' % confidence)
+                logger.info('Validating in Browser')
```
Comment (s0md3v) [inline modes/scan.py]: Unnecessary output

Comment (s0md3v) [inline modes/scan.py]: Unnecessary output

File: core/browserEngine.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    try:
+        actions.move_by_offset(2, 2)
+        actions.perform()
+        if EC.alert_is_present():
+            popUp = True
+            logger.good('Found alert during browser validation')
+
+    except UnexpectedAlertPresentException:
+        popUp = True
+        logger.good('Found alert during browser validation')
```
Comment (s0md3v) [inline core/browserEngine.py]: Unnecessary output

File: core/browserEngine.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    os.remove('test.html')
+    popUp = False
+    actions = webdriver.ActionChains(browser)
+
+    try:
+        actions.move_by_offset(2, 2)
+        actions.perform()
+        if EC.alert_is_present():
+            popUp = True
+            logger.good('Found alert during browser validation')
```
Comment (s0md3v) [inline core/browserEngine.py]: Unnecessary output

File: modes/scan.py (hunk 1 above)
Comment (KalibRx) [reply modes/scan.py]: Removed

Comment (KalibRx) [reply modes/scan.py]: Removed

File: core/browserEngine.py (hunk 2 above)
Comment (KalibRx) [reply core/browserEngine.py]: Removed

**Pull request merged**